### ISSUE TITLE
Gitlab integration stopped showing clockify tracker widget in PR overview

### ISSUE THREAD

**Issue by mbalc** [clockify/browser-extension#218]
https://gitlab.com/[ORG]/[PROJECT]/-/merge_requests/[PR_ID]

![image](https://user-images.githubusercontent.com/11033790/193079808-c92885de-e727-4c79-8449-9b4b0f0b0cd3.png)

no errors are showing in the dev console

**Comment by 7saraK** [comment]
Hi @mbalc, thank you for letting us know this. 
Can you please advise which version of the Clockify extension you have installed? 

Have you tried uninstalling > clearing cache/cookies > reinstalling again? 

Please let me know. Cheers!

**Comment by Bobspadger** [comment]
I am also having this issue with a custom domain gitlab install.

I've removed the extension, clear cookies and cache, re-installed and re-configured and no links in the browser are appearing to start and stop timers

**Comment by Eusebius1920** [comment]
Same here:

custom gitlab domain + chromium --> nothing appears

**Comment by 7saraK** [comment]
Thank you all for confirming this. I'll forward the issue to the team, and will reach you back here once the fix is released. Cheers!

**Comment by 7saraK** [comment]
Hi all, can you please advise if your gitlab custom domain has been added to the Clockify extension settings? 
Please see the steps to add it: 

Open extension
Open settings (hamburger menu in the upper left corner)
Click Integrations
Scroll down and under Custom domains enter your domain (e.g. mydomain.com)
Select the tool that’s on that domain
Click Add

**Comment by Eusebius1920** [comment]
It indeed was added and I checked it. It just stopped working at one point.

Removing and re-adding **did** help. (I did not check immediatley before removing and re-adding, if it still did not work or if something made it working without my doing)

The domain was added and then re-added only with the domain (not the schema https://)

**Comment by Bobspadger** [comment]
Yes, I have been having to reset clockify and re-add it. It then works for a bit, and often just stops.

**Comment by 7saraK** [comment]
Thank you for the comment @Bobspadger. I'll let the team know, and will update here accordingly.

**Comment by mbalc** [comment]
> Hi @mbalc, thank you for letting us know this. Can you please advise which version of the Clockify extension you have installed?
> 
> Have you tried uninstalling > clearing cache/cookies > reinstalling again?
> 
> Please let me know. Cheers!

I'm not sure which version I used back then

However, I can confirm that on version 2.8.4 it now seems to be working for me once again
